src/plugins/intel_npu/src/al/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME ov_npu_al)
```

Comment:
let's restore `openvino_npu_al` and add extra line https://github.com/openvinotoolkit/openvino/blob/master/src/common/conditional_compilation/CMakeLists.txt#L10